gptables/examples/penguins_multiple_sheets.py

```diff
@@ -0,0 +1,58 @@
+"""
+Penguins - Cover Page
+-----------------
+
+This example demonstrates how to create a workbook with multiple sheets.
```

Comment:
Consider adding a comment to explain that a table of contents will be autogenerated